sopel/config/types.py

```diff
@@ -639,16 +640,5 @@ def parse(self, value, main_config, this_section):
                 raise ValueError("Value must be an existing or creatable file.")
         return value
 
-    def serialize(self, value, main_config, this_section):
-        """Used to validate ``value`` when it is changed at runtime.
-
-        :param main_config: the config object which contains this attribute
-        :type main_config: :class:`~sopel.config.Config`
-        :param this_section: the config section which contains this attribute
-        :type this_section: :class:`~StaticSection`
-        :return: the ``value``, if it is valid
-        :rtype: str
-        :raise ValueError: if the ``value`` is not valid
-        """
```

Comment:
Hmm, other `serialize` methods do have a docstring describing their purpose relative to that class. In this case we don't want the one from `BaseValidated` to get inherited, because it doesn't document args and says "must be implemented in subclasses" (which doesn't make sense, because it _is_ implemented and this _is_ a subclass).

Reply:
Fixed it.